NtupleProducer/plugins/EleFiller.cc

```diff
@@ -70,28 +70,13 @@ class EleFiller : public edm::EDProducer {
   edm::EDGetTokenT<vector<Vertex>> theVtxTag ;
   int sampleType;
   int setup;
-  const StringCutObjectSelector<pat::Electron, true> cut;
-  const CutSet<pat::Electron> flags;
+  //const StringCutObjectSelector<pat::Electron, true> cut;
```

Comment:
I think this "cut" should be kept. It comes from:
https://github.com/LLRCMS/LLRHiggsTauTau/blob/94X_HH/NtupleProducer/test/analyzer.py#L47